test/readability.js

```diff
@@ -12,13 +13,11 @@ describe('readability', () => {
       test: 'whateverfake!'
     });
     results.errors.should.have.length(1);
-    results.errors[0].stack.should.eql(
-      'instance.test is not exactly one from </RequestSchema>,</FunctionSchema>'
-    );
+    results.errors[0].stack.should.eql('instance is not of a type(s) object');
   });
 
   it('should have decent messages for minimum length not met', () => {
-    const results = CreateSchema.validate({
+    const results = TriggerSchema.validate({
```

Comment:
Any reasons why this change?

Reply:
Triggerschema operation is itself an `anyOf` (polling/hook) so I wanted to make sure errors in a nested field  (like `inputData`) correctly surfaces in that situation. Sort of the base case we're trying to handle